test/test_collector.py

```diff
@@ -107,7 +107,7 @@ def env_fn(seed):
             break
     with pytest.raises(AssertionError):
         assert_allclose_td(b1, b2)
-    collector.shutdown()
```

Comment:
can we keep these, as we test it somewhere else?
Like this if for some reason the new feature breaks only that test will fail

Reply:
Sounds good